### PR TITLE
CLIP-1734: Make skip refresh optional

### PR DESCRIPTION
**Pull request by bianchi2**
While https://github.com/atlassian-labs/data-center-terraform/pull/291 fixes the error when destroying terraform after a failed terraform apply, it introduces another issue. Skipping refresh can sometimes make Helm provider keep using an invalidated token, which results in the following error:

```
Error: Kubernetes cluster unreachable: the server has asked for the client to provide credentials
```
This PR makes `-refresh=false` optional, and it can be added to terraform destroy args with `-s` flag. 2 additional FAQs were added as well.

## Checklist
- [x] I have successful end to end tests run (with & without domain)
- [ ] I have added unit tests (if applicable)
- [x] I have user documentation (if applicable)

### CODE REVIEW

File: docs/docs/troubleshooting/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 This guide contains general tips on how to investigate an application deployment that doesn't work correctly.
 
-??? tip "How do I uninstall an environment using a different Terraform configuration file?" 
-
```
Comment (nghazali) [inline docs/docs/troubleshooting/TROUBLESHOOTING.md:6]: There are lot's of unnecessary changes comes from your editor settings which makes the PR very noisy, and because all other have different setting, probably if anyone else makes another change in the files they will repeat this noise again. I suggest change the rules on your editor to avoid those changes. I see yours end trims all spaces which means change in many lines.

Comment (bianchi2) [reply docs/docs/troubleshooting/TROUBLESHOOTING.md]: Fixed by editing in GitHub. Somehow Atom would trim those spaces

**Review by nghazali — APPROVED** [review]

File: uninstall.sh [hunk 2]
```diff
@@ -118,7 +124,7 @@ destroy_infrastructure() {
     terraform -chdir="${ROOT_PATH}" init -no-color | tee -a "${LOG_FILE}"
   fi
   set +e
-  terraform -chdir="${ROOT_PATH}" destroy -auto-approve -refresh=false -no-color "${OVERRIDE_CONFIG_FILE}" | tee -a "${LOG_FILE}"
+  terraform -chdir="${ROOT_PATH}" destroy -auto-approve ${SKIP_REFRESH} -no-color "${OVERRIDE_CONFIG_FILE}" | tee -a "${LOG_FILE}"
```
[the remark this answers is not in the document]
Comment (jjeongatl) [reply uninstall.sh:127]: for the sake of consistency
